sylt-compiler/src/typechecker.rs

```diff
@@ -631,7 +625,10 @@ impl TypeChecker {
             | StatementKind::Blob { .. }
             | StatementKind::IsCheck { .. }
             | StatementKind::ExternalDefinition { .. } => {
-                unreachable!("Illegal inner statement! Parser should have caught this.")
+                unreachable!(
+                    "Illegal inner statement at {:?}! Parser should have caught this.",
+                    span
```

Comment:
Isnt this debug print is really bad? Maybe say which line and rsnge it has instead?

Reply:
kinda bad, yes, but i don't know how to get the filename from just the span.

Reply:
You can check the type_error macro - the span has a `file_id` that indexes `namespace_to_filename` (I think these names are right, I might be wrong.)